examples/fungible-token/src/index.js

```diff
@@ -0,0 +1,78 @@
+import {NearContract, NearBindgen, call, view, near} from '../../../sdk'
+import {LookupMap} from '../../../sdk/collections/lookup-map'
+
+function assert(b, str)
+{
+    if (b) {
+        return
+    } else {
+        throw Error("assertion failed: " + str)
```

Comment:
Host panic works. We use `throw new Error` is because more idiomatic in JavaScript. It is then converted to host panic call in `jsvm.c` so the effect is same.